packages/core/content-manager/admin/src/pages/ListView/components/TableActions.tsx

```diff
@@ -52,9 +52,10 @@ const TableActions = ({ document }: TableActionsProps) => {
   return (
     <DescriptionComponentRenderer
       props={props}
-      descriptions={(
-        plugins['content-manager'].apis as ContentManagerPlugin['config']['apis']
-      ).getDocumentActions()}
+      descriptions={(plugins['content-manager'].apis as ContentManagerPlugin['config']['apis'])
+        .getDocumentActions()
+        // We explicitly remove the PublishAction from description so we never render it and we don't make unnecessary requests.
+        .filter((action) => action.name !== 'PublishAction')}
```

Comment:
why we don't need bulk publish?

Reply:
This is not related to Bulk Publish. Is about the individual publish button which is not showed on the ListView, so here we avoid run the component